snapcraft.yaml

```diff
@@ -1500,6 +1516,18 @@ parts:
     prime:
       - bin/setup-shmounts
 
+  gpu-2404:
```

Comment:
Does this not get provided by the connected gpu-2404 snap? Whyy do we need this pulled here too?

Reply:
no, this is not provided by `gpu-2404`. This repo (https://github.com/canonical/gpu-snap) contains tools to help integration with `gpu-2404` interface snaps. This step, specifically, needed to clean up libraries on the prime step to prevent duplicates.

Reply:
You can read about it in the official guide https://mir-server.io/docs/the-gpu-2404-snap-interface